core-bundle/src/Command/DebugFragmentsCommand.php

```diff
@@ -42,7 +42,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         foreach ($fragments as $identifier => $config) {
             $class = new \ReflectionClass(AbstractFragmentController::class);
-            $attributes = $class->getProperty('options')->getValue($this->container->get($config->getController()));
+
+            try {
```

Comment:
should we check for the interface here as well? or something like `hasProperty()`?

Reply:
Instead of catching the exception?